teams/leadership-council.toml

```diff
@@ -3,15 +3,15 @@ name = "leadership-council"
 [people]
 leads = []
 members = [
-    "carols10cents",
-    "eholk",
-    "ehuss",
-    "jackh726",
-    "jonathanpallant",
-    "Mark-Simulacrum",
-    "m-ou-se",
-    "rylev",
-    "technetos",
+    { github = "carols10cents", roles = ["council-rep-crates-io"] },
+    { github = "eholk", roles = ["council-rep-compiler"] },
+    { github = "ehuss", roles = ["council-rep-devtools"] },
+    { github = "jackh726", roles = ["council-rep-lang"] },
+    { github = "jonathanpallant", roles = ["council-rep-launching-pad"] },
+    { github = "Mark-Simulacrum", roles = ["council-rep-release"] },
+    { github = "m-ou-se", roles = ["council-rep-libs"] },
+    { github = "rylev", roles = ["council-rep-infra"] },
+    { github = "technetos", roles = ["council-rep-mods"] },
```

Comment:
Data is from https://blog.rust-lang.org/2023/06/20/introducing-leadership-council.html.

Reply:
Also #1058 / #1059.